### PR TITLE
Adds keymap for muzfuz/lunar

### PR DESCRIPTION
**Pull request by muzfuz**
Adds keymap for `muzfuz/lunar`

### CODE REVIEW

File: keyboards/ai03/lunar/keymaps/muzfuz/readme.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# muzfuz's keymap for Lunar
+
+```shell
+make ai03/lunar:muzfuz
```
Comment (drashna) [inline keyboards/ai03/lunar/keymaps/muzfuz/readme.md]: The `:dfu` target should work for this, actually: 

```suggestion
make ai03/lunar:muzfuz:dfu
```

Comment (muzfuz) [reply keyboards/ai03/lunar/keymaps/muzfuz/readme.md]: Great!

File: keyboards/ai03/lunar/keymaps/muzfuz/readme.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# muzfuz's keymap for Lunar
+
+```shell
+make ai03/lunar:muzfuz:dfu
+dfu-programmer atmega32u4 erase --force && \
```
Comment (drashna) [inline keyboards/ai03/lunar/keymaps/muzfuz/readme.md]: If you have the above, you shouldn't need the rest.  

If you want to leave it in here, that's fine,  though I'd say use a different section/block for that, then.

Comment (muzfuz) [reply keyboards/ai03/lunar/keymaps/muzfuz/readme.md]: Cheers, that makes sense :) I've pushed that change.

**Comment by drashna** [comment]
Thanks!

**Comment by muzfuz** [comment]
Thank you @drashna 💯

**Pull request merged**